### PR TITLE
Flesh out issues, clean up wording

### PR DESCRIPTION
**Pull request by ccameron-chromium**
Several issues have been accumulating at the bottom of the document. Provide explanation for the resolution of those issues.

Perform some clean-up to wording in earlier sections referred to by the issues section.

### CODE REVIEW

**Comment by ccameron-chromium** [comment]
This solidifies the Issues section.

I added an issue about attaching the color space to the CanvasRenderingContext2D to ventilate the issue that we discussed in the last meeting.

File: CanvasColorSpaceProposal.md [hunk 1]
```diff
@@ -95,13 +96,16 @@ partial dictionary CanvasRenderingContext2DSettings {
 };
 </pre>
 
-* All input colors (e.g, ``fillStyle`` and ``strokeStyle``) follow the same interpretation as CSS color literals, regardless of the canvas color space.
-* Images drawn to a canvas are converted from their native color space to the backing storage color space of the canvas.
+2D Canvases are color managed.
+* All inputs that are drawn to a canvas have specific color space
```
Comment (kenrussell) [inline CanvasColorSpaceProposal.md:100]: "have a specific color space" - and end sentence with period - but see comment below.

File: CanvasColorSpaceProposal.md [hunk 2]
```diff
@@ -95,13 +96,16 @@ partial dictionary CanvasRenderingContext2DSettings {
...
+  * Input colors (e.g, ``fillStyle`` and ``strokeStyle``) follow the same interpretation as CSS color literals, regardless of the canvas color space.
   * Images with no color profile, when drawn to the canvas, are assumed to be in the sRGB color space.
-* All blending operations are performed in the canvas' backing storage color space.
+* All inputs that are drawn to a canvas are converted to the canvas' color space before being drawn to the canvas.
```
Comment (kenrussell) [inline CanvasColorSpaceProposal.md:103]: Does this make the first bullet point obsolete?

File: CanvasColorSpaceProposal.md [hunk 3]
```diff
@@ -202,21 +206,91 @@ The ``getImageData`` method is responsible for converting the data from the canv
...
 </pre>
 
-## Adoption
-Lack of color management and color interoperability is a longstanding complaint about the canvas API.
-Authors of games and imaging apps are expected to be enthusiastic adopters.
+## Resolved Issues
+
+### Additional color spaces, high bit depth, and high dynamic range
+
+Should we support color spaces besides `'srgb'` and `'display-p3'`.
```
Comment (kenrussell) [inline CanvasColorSpaceProposal.md:213]: End sentence with "?".

File: CanvasColorSpaceProposal.md [hunk 4]
```diff
@@ -202,21 +206,91 @@ The ``getImageData`` method is responsible for converting the data from the canv
...
+We defer supporting color spaces that require more than 8 bit of precision to a separate proposal that also covers high bit depth and high dyanmic range.
+
+### Custom color profile support
+
+Should we support custom color spaces based on ICC profiles?
+
+Not now.
+Rendering to an arbitrary ICC profile adds a significant level of complexity to the implementation.
+This proposal does not add this capability, but does not close the door to its addition in a future proposal.
+The ``CanvasColorSpaceEnum`` could be change to a ``DOMString`` that could refer any CSS color profile, including CSS Color Module Level 4's [custom profiles](https://www.w3.org/TR/css-color-4/#at-profile).
```
Comment (kenrussell) [inline CanvasColorSpaceProposal.md:227]: change -> changed

File: CanvasColorSpaceProposal.md [hunk 5]
```diff
@@ -202,21 +206,91 @@ The ``getImageData`` method is responsible for converting the data from the canv
...
+* If context creation were to throw on an unrecognized attribute, then this code would fail only on browsers that support canvas color spaces.
+
+Succeeding on a browser that does not support the feature and failing on a browser that does support the feature is an undesirable behavior.
+
+### Naming of color gamuts
+
+The [Media Query APIs](https://www.w3.org/TR/mediaqueries-4/) use the name "p3", while the [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/#predefined) uses the name "display-p3". Could this divergence be confusing?
+
+Yes and no.
+Indeed it can be confusing that a gamut name and color space name similar but different.
```
Comment (kenrussell) [inline CanvasColorSpaceProposal.md:255]: name similar -> name are similar

File: CanvasColorSpaceProposal.md [hunk 6]
```diff
@@ -202,21 +206,91 @@ The ``getImageData`` method is responsible for converting the data from the canv
...
+Note that WebGL puts the ``colorSpace`` on the ``WebGLRenderingContextBase`` interface.
+But, changing the WebGL color space just causes the pixels to be reinterpreted, but that is suitable because WebGL is not color managed, and raw pixel access is the norm.
 
-* Should there be API-level support for mixing chromaticities and transfer functions, including use of no-op transfer functions?
+In contrast, a 2D canvas is color managed and does not allow raw pixel access.
+What should happen to the canvas if its color space changes?
+There are three options:
 
-* Should context creation throw on an unrecognized, non-undefined creation attribute?
+* Destroy the context's contents. This is like what happens when a ``HTMLCanvasElement`` is resized.
```
Comment (kenrussell) [inline CanvasColorSpaceProposal.md:289]: Suggest just: "This is what happens..."

**Review by kenrussell — APPROVED** [review]
High level structure looks good - a few editorial comments.

File: CanvasColorSpaceProposal.md (hunk 6 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md:289]: Fixed.

File: CanvasColorSpaceProposal.md (hunk 5 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md:255]: changed to "are similar but not the same".

File: CanvasColorSpaceProposal.md (hunk 4 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md:227]: Done.

File: CanvasColorSpaceProposal.md (hunk 3 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md:213]: Fixed

File: CanvasColorSpaceProposal.md (hunk 2 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md:103]: I changed it from "are converted to the canvas' color space" to "are converted from their color space to the canvas' color space". I wanted to make the points separately that there are no color-space-less inputs to a canvas, and that the conversion is done before drawing. I added a parenthetical about no-color-space-less inputs.

File: CanvasColorSpaceProposal.md (hunk 1 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md:100]: Fixed

**Review by kenrussell — APPROVED** [review]
Thanks for the updates; looks good to me!

**Review by jdashg — APPROVED** [review]

**Pull request merged**